src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -552,7 +570,8 @@ export default {
           name: FilingTypes.CHANGE_OF_ADDRESS,
           certifiedBy: this.certifiedBy || '',
           email: 'no_one@never.get',
-          date: this.currentDate
+          date: this.currentDate, // NB: API will reassign this date according to its clock
+          effectiveDate: this.convertLocalDateToUTCDateTime(this.coaDate)
```

Comment:
Effective Date is not used atm, but now this filing is consistent with AR and COD.